src/stravalib/model.py

```diff
@@ -410,7 +410,8 @@ class AthleteStats(strava_model.ActivityStats):
 
 
 class MetaAthlete(strava_model.MetaAthlete, BoundClientEntity):
-    pass
+    # Undocumented
+    resource_state: Optional[int]
```

Comment:
does anyone every use / need this? i see it throughout . i'm guessing the answer is no?